src/pages/dashboard/Explore.tsx

```diff
@@ -113,6 +113,8 @@ const Explore = () => {
       const resourceTypes = searchTerms.getAll(SearchParameters.resourceType);
       const tags = searchTerms.getAll(SearchParameters.tag);
       const pageTerm = searchTerms.get(SearchParameters.page);
+      const showInaccessibleParameter = searchTerms.get(SearchParameters.showInaccessible);
+      const showInaccessible = showInaccessibleParameter ? showInaccessibleParameter.includes('true') : false;
```

Comment:
to lowercase ?
kanskje også bruke ===   (ellers kan jo 'not true' også være gyldig ;-)

Reply:
Fikset